pyproject.toml

```diff
@@ -19,7 +20,7 @@ packages = [
 
```

Comment:
I think we have to add the data files in here somewhere right? Otherwise they won't be installed by pip?

Reply:
Tested via PyPI test and works fine :)